crates/cli-support/src/js/binding.rs

```diff
@@ -1417,7 +1419,23 @@ impl Invocation {
     }
 }
 
-fn adapter2ts(ty: &AdapterType, dst: &mut String) {
+#[derive(Debug, Clone, Copy)]
+enum NoneType {
+    Undefined,
+    Null,
```

Comment:
This is never built, right?

Instead, why don't we do something like this:
```rust
pub enum TypePosition {
    Argument,
    Return,
}
```

Reply:
I used to use it, but not anymore after some code changes before I made the PR.

But your suggestion is better. I'm planing to do more changes that make inputs more accepting in the future, so this is perfect.